core/runtime/TRTEngine.cpp

```diff
@@ -196,7 +209,6 @@ TRTEngine::TRTEngine(
 }
 
 TRTEngine::~TRTEngine() {
-  cudagraph.reset();
```

Comment:
Why is this not needed ?

Reply:
The object destructor calls it and its potentially unsafe to do it ourselves